_includes/header.html

```diff
@@ -1,47 +1,33 @@
 <header class="site-header">
-
-  <nav class="navbar navbar-default navbar-static-top site-nav" role="navigation">
+  <nav class="navbar navbar-expand-md site-nav">
     <div class="container">
-      <div class="row">
-        <div class="col-md-10 col-xm-12 block-center">
-          <div class="navbar-wrapper">
-          <div class="navbar-header">
-            <a href="#" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse">
-              <span class="sr-only">Toggle navigation</span>
-              <div class="icon-bars">
-                <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
-              </div>
-
-              <span class="menu-label">MENU</span>
-            </a>
-            <a class="navbar-brand" href="{{ site.baseurl }}/">
-              <img class="header-logo" src="{{site.baseurl}}/img/indico-blue-white.png">
-            </a>
-          </div>
+      <a class="navbar-brand" href="{{ site.baseurl }}">
+        <img class="header-logo" src="{{ site.baseurl }}/img/logos/indico_white.png">
+      </a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-collapse" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon">
+            <img src="{{ site.baseurl }}/img/hamburger-menu.png">
```

Comment:
too much indentation

Reply:
ok looks like we have files with 4 and 2 spaces... can we add a `.editorconfig` and then reformat it consistently? (ideally after merging the PR to avoid tons of whitespace changes in the diff here)